### ISSUE TITLE
Remove or change input option alias for the number of processes

### ISSUE THREAD

**Issue by andreas-gruenwald** [webmozarts/console-parallelization#31]
``p`` is currently used as the alias for the number of concurrent processes that are started by the trait:
https://github.com/webmozarts/console-parallelization/blob/master/src/Parallelization.php#L139

``p`` is also a quite commonly used input option shortcut (parent, part, ...) for existing commands. 

Please reconsider to pick another, less commonly used shortcut or just the remove the shortcut.

**Comment by theofidry** [comment]
>p is also a quite commonly used input option shortcut (parent, part, ...) for existing commands.

Why is it a problem though? The risk here is conflicts with the other command aliases and it's not because there is a shortcut that you must use it either.

**Comment by andreas-gruenwald** [comment]
I tried to implement the Parallelization Trait for an existing command, and this command already had ``p`` in use. As I am trying to integrate the trait into a framework (Pimcore), we need to be downward compatible and cannot replace the old shortcut by the new one.

Actually the problem can be resolved by overriding the ``configureParallelization()`` method.
This is not very convenient, but it works. As longs as we have got this option I am fine.
However I would prefer to have another option to disable / configure that shortcut.
